cmd/receiver-proxy/main.go

```diff
@@ -227,10 +240,20 @@ func runMain(cCtx *cli.Context) error {
 
 	maxUserRPS := cCtx.Int(flagMaxUserRPS)
 
+	certDuration := cCtx.Duration("cert-duration")
+	certHosts := cCtx.StringSlice("cert-hosts")
+	certPath := cCtx.String(flagCertPath)
+	certKeyPath := cCtx.String(flagCertKeyPath)
+	if certPath == "" || certKeyPath == "" {
```

Comment:
Hm, so this would be a breaking change
Would it not make sense to run with ephemeral certs if those are empty?
Either both of them should be set or both should be empty in that case

Reply:
i figured we'd make it required persistent, as it's the mode it should always run in 🤔